### DISCUSSION TITLE
Copilot does not create code from comments [Java, IntelliJ]

### DISCUSSION THREAD

**Discussion by JustLutz** [community/community#17649] [Copilot Conversations]
I installed GitHub Copilot today and started following the "get started" docs. It works well, but when it says, that Copilot can create code from commands and I tried it with the given example the copilot doesn´t suggest a codeblock. Does it work for you and why it don´t work for me?

**Comment by jansorg** [comment]
Thanks for reporting! Could you post the snippet with the comment you tried to use?
Please make sure that you're on the latest version (1.1.24.1610).
